src/Prometheus/Storage/Redis.php

```diff
@@ -51,6 +56,19 @@ public function __construct(array $options = [])
         $this->redis = new \Redis();
     }
 
+    public static function fromExistingConnection(\Redis $redis): self
+    {
+        if (false === $redis->isConnected()) {
```

Comment:
we're using non-yoda style comparisons in the rest of the codebase. Please change this to be `actual === expected`

Reply:
This still needs changing

Reply:
It is fixed, you are looking at outdated change.